cg/services/pdc_service/pdc_service.py

```diff
@@ -86,57 +86,6 @@ def run_dsmc_command(self, command: list) -> None:
                 return
             raise PdcError(message=f"{error}") from error
 
-    def validate_is_flow_cell_backup_possible(
-        self, db_flow_cell: Flowcell, flow_cell_encryption_api: FlowCellEncryptionAPI
-    ) -> bool:
-        """Check if back-up of flow cell is possible.
-        Raises:
-            DsmcAlreadyRunningError if there is already a Dsmc process ongoing.
-            FlowCellAlreadyBackupError if flow cell is already backed up.
-            FlowCellEncryptionError if encryption is not complete.
-        """
-        if self.validate_is_dsmc_running():
-            raise DsmcAlreadyRunningError("Too many Dsmc processes are already running")
-        if db_flow_cell and db_flow_cell.has_backup:
-            raise FlowCellAlreadyBackedUpError(
-                f"Flow cell: {db_flow_cell.name} is already backed-up"
-            )
-        if not flow_cell_encryption_api.complete_file_path.exists():
-            raise FlowCellEncryptionError(
-                f"Flow cell: {flow_cell_encryption_api.flow_cell.id} encryption process is not complete"
-            )
-        LOG.debug("Flow cell can be backed up")
-
-    def backup_flow_cell(
-        self, files_to_archive: list[Path], store: Store, db_flow_cell: Flowcell
-    ) -> None:
-        """Back-up flow cell files."""
-        for encrypted_file in files_to_archive:
-            if not self.dry_run:
-                self.archive_file_to_pdc(file_path=encrypted_file.as_posix())
-        if not self.dry_run:
-            store.update_flow_cell_has_backup(flow_cell=db_flow_cell, has_backup=True)
-            LOG.info(f"Flow cell: {db_flow_cell.name} has been backed up")
-
-    def start_flow_cell_backup(
-        self,
-        db_flow_cell: Flowcell,
-        flow_cell_encryption_api: FlowCellEncryptionAPI,
-        status_db: Store,
-    ) -> None:
-        """Check if back-up of flow cell is possible and if so starts it."""
-        self.validate_is_flow_cell_backup_possible(
-            db_flow_cell=db_flow_cell, flow_cell_encryption_api=flow_cell_encryption_api
-        )
-        self.backup_flow_cell(
-            files_to_archive=[
-                flow_cell_encryption_api.final_passphrase_file_path,
-                flow_cell_encryption_api.encrypted_gpg_file_path,
-            ],
-            store=status_db,
-            db_flow_cell=db_flow_cell,
-        )
-
```

Comment:
most important part here: moving of these backup related frunctions out of the pdc service. So that it only is used for pdc related commands

Reply:
The `IlluminaBackupService` should make these function calls and make use of the pdc service.